Fastfile.private

```diff
@@ -283,6 +274,28 @@ reflected in `Info.plist` during building."
 
   end
 
+  desc "Refreshes the `Certificates` and `Provisioning Profiles`."
```

Comment:
I'd clear it up saying "Updates or downloads" instead of "Refreshes". Name is fine, but I think this description help a little more with that.